tracing-attributes/tests/err.rs

```diff
@@ -205,6 +206,36 @@ fn test_err_dbg() {
     handle.assert_finished();
 }
 
+#[instrument(err(Value))]
+fn err_value() -> Result<u8, Box<dyn Error + 'static>> {
+    Err("oh no".into())
+}
+
+#[test]
+fn test_err_raw() {
+    let err: Box<dyn Error + 'static> = "oh no".into();
+    let span = expect::span().named("err_raw");
```

Comment:
this test will fail now that the name of the instrumented function has been changed from `err_raw` to `err_value`:
```suggestion
    let span = expect::span().named("err_value");
```